client/src/Views/ReportsPage/UserReports.vue

```diff
@@ -25,15 +25,15 @@
         class="user-container__users-list"
         :data="usersList"
         :loading="loading"
-        :restricted="{ footer: 'disable-items' }"
         :searchable="true"
         :selectable="true"
         @selectedUser="handleSelectedUser"
         title="Users"
       />
-      <div v-if="user.id">
+      <div v-if="user.id" class="user-container__profile-card">
         <UserProfileCard :user="user" />
       </div>
+      <div v-else class="user-container__profile-card--placeholder"></div>
```

Comment:
Nice 👍

Reply:
Had to on the larger screen sizes or it wouldn't cooperate